fluentd-gcp-scaler/scaler.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/sh
+
+SLEEP_SECONDS=${SLEEP_SECONDS:-60}
+NAMESPACE=${NAMESPACE:-kube-system}
+DEFAULT_CPU_REQUEST=${CPU_REQUEST}
```

Comment:
Is it intended to be "" if not specified? Would it be more explicit to write DEFAULT_CPU_REQUEST=${CPU_REQUEST:-}

Reply:
Done.